src/Compilers/VisualBasic/Portable/Parser/BlockContexts/CompilationUnitContext.vb

```diff
@@ -4,6 +4,8 @@ Imports Microsoft.CodeAnalysis.Syntax.InternalSyntax
 Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax.FeatureUtils
```

Comment:
InternalSyntax is a poor place for these helpers. That namespace is exclusively for implementation details of the internals of the parse tree.